docs/guides/states.md

```diff
@@ -217,6 +219,52 @@ const canGo = state.hasTag('go');
 // => `true` if in 'green' or 'yellow' state
 ```
 
+### `state.can(event)`
+
+_Since 4.25.0_
+
+This method determines whether an `event` will cause a state change if sent to the interpreted machine. It will return `true` if the state will change due to the `event` being sent; otherwise `false`:
```

Comment:
can we get a reference here as to what we mean by a state change?
https://xstate.js.org/docs/guides/states.html#state-changed

Reply:
```suggestion
This method determines whether an `event` will cause a change in state (or context) if sent to the interpreted machine. It will return `true` if the state will change due to the `event` being sent; otherwise `false`:
```